src/main/java/com/segment/analytics/android/integrations/comscore/ComScoreIntegration.java

```diff
@@ -221,61 +223,66 @@ private void trackVideoPlayback(
     playbackMapper.put("sound", "ns_st_vo");
 
     Map<String, String> playbackAsset =
-        buildPlaybackAsset(properties, comScoreOptions, playbackMapper);
+            buildPlaybackAsset(properties, comScoreOptions, playbackMapper);
 
     if (name.equals("Video Playback Started")) {
       streamingAnalytics = comScoreAnalytics.createStreamingAnalytics();
       streamingAnalytics.createPlaybackSession();
-      streamingAnalytics.setLabels(playbackAsset);
+      streamingAnalytics.getConfiguration().addLabels(playbackAsset);
 
       // The label ns_st_ci must be set through a setAsset call
       Map<String, String> contentIdMapper = new LinkedHashMap<>();
       contentIdMapper.put("assetId", "ns_st_ci");
 
       Map<String, String> contentIdAsset = mapSpecialKeys(properties, contentIdMapper);
```

Comment:
A little nitpicky, but I think `contentIdAsset` variable name is a bit unclear. Can we change it to something like `mappedContentProperties` or `mappedContentProps`?

Reply:
I am on it.